pkg/gui/gui.go

```diff
@@ -583,11 +583,28 @@ func (gui *Gui) Run(filterPath string) error {
 	}
 
 	gui.waitForIntro.Add(1)
-	if gui.c.UserConfig.Git.AutoFetch {
-		go utils.Safe(gui.startBackgroundFetch)
+
+	if userConfig.Git.AutoFetch {
+		fetchInterval := userConfig.Refresher.FetchInterval
+		if fetchInterval > 0 {
```

Comment:
Any opinion on whether this check should be made here or in `startBackgroundFetch()`? Keeping it here means we have both interval checks at the same place - moving it to the function would mean the caller doesn't need to ensure the interval is OK before invoking it.

Reply:
I'm happy with the way this is now